notebooks/04_filter.ipynb

```diff
@@ -0,0 +1,1004 @@
+{
```

Comment:
<p>Can we show evidence that we removed <code>width &lt; height</code> images to users in this example? I think that we cannot see because it just shows the number of images before and after.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/datumaro/pull/748/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
I added `get_width_height` function to show width and height for each dataset item.